organisations/models.py

```diff
@@ -56,10 +57,20 @@ def delete(self):
         return super().delete()
 
 
+class UserQueryset(models.QuerySet):
```

Comment:
There is now something that affects to the admin login. The admin-login with this PR code is not currently possible. If I use the production version't commit to login and then change to this PR branch after successful login, all seems to work OK with the logged in user.

![image](https://user-images.githubusercontent.com/389204/210203876-7a2720a4-6e79-4095-960a-cd30461beb2b.png)

Reply:
@tuomas777 suggested a fix like this and I tested that it seems to work:
```
from django.contrib.auth.models import UserManager
class User(AbstractUser):
    objects = UserManager.from_queryset(UserQueryset)()
```

Reply:
Added that fix to the code.